content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -537,6 +536,20 @@ spec:
       ...
 ```
 
+You can also toggle Job suspension by patching the Job using the command line.
+
+Suspend an active Job:
+
+```shell
+kubectl patch job/myjob -p '{"spec":{"suspend":true}}'
```

Comment:
In docs, I like to write things longhand (it helps the reader);
```suggestion
kubectl patch job/myjob --type=strategic --patch '{"spec":{"suspend":true}}'
```

Now the reader can immediately see that this is a strategic merge patch.

Reply:
This is great @sftim - Thank you.  I've committed another change so both examples are in sync.